recipes/libblst/meta.yaml

```
@@ -0,0 +1,129 @@
+{% set name = "libblst" %}
+{% set version = "0.3.13" %}
+
+{% set so_major_version = version.split('.')[0] %}
+package:
+  name: libblst-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/supranational/blst/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 89772cef338e93bc0348ae531462752906e8fa34738e38035308a7931dd2948f
+  patches:
+    - patches/0001-unix-add-libversion.patch  # [unix]
+    - patches/0001-win-simplify-build.sh.patch  # [win]
+
+build:
+  number: 0
+  script_env:
+    - PKG_MAJOR_VERSION={{ so_major_version }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}  # [not win]
+    - {{ compiler('m2w64_c') }}  # [win]
+    - {{ stdlib('c') }}  # [not win]
+    - {{ stdlib('m2w64_c') }}  # [win]
+    - libxcrypt  # [unix]
+    - llvm-tools  # [win]
+    - sed  # [unix]
+
+outputs:
+  - name: libblst
```

Let's call it `-devel` as that is the common name we use for splitted libraries.
```suggestion
  - name: libblst-devel
```